### PR TITLE
add new sample code for batch processing

### PR DESCRIPTION
**Pull request by kaismiin**

### CODE REVIEW

File: sample2.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import time
+
+try:
+    import thread
+except ImportError:
+    import _thread as thread
+
+SAMPLE_RATE = 16000
+CHUNK_SIZE = 5760
+SLEEP_TIME = CHUNK_SIZE / (SAMPLE_RATE * 2.0)
```
Comment (simonsonga) [inline sample2.py]: Explain this formula

File: sample2.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
+from aicsapi_speech_recognition import SpeechRecognitionApi
```
Comment (simonsonga) [inline sample2.py]: rename to batchSample.py ?

File: wav.scp [hunk 3]
```diff
@@ -0,0 +1,2 @@
+a audio/a.wav
```
Comment (simonsonga) [inline wav.scp]: Why need this file ?

Comment (kaismiin) [reply wav.scp]: I already rename it

File: sample2.py (hunk 2 above)
Comment (kaismiin) [reply sample2.py]: fixed

**Pull request merged**